umake/frameworks/ide.py

```diff
@@ -629,7 +629,7 @@ def __init__(self, category, flavour=""):
                          download_page="https://netbeans.org/downloads/zip.html",
                          dir_to_decompress_in_tarball="netbeans*",
                          desktop_filename="netbeans{}.desktop".format(flavour),
-                         packages_requirements=['openjdk-7-jdk', 'jayatana'],
```

Comment:
We removed it from the intellij ides recently because it was actually disabled in the exec sh file.
I'm not sure what's going on with jayatana.
Seems like removing it is the best thing to do now.